msteams-platform/graph-api/rsc/grant-resource-specific-consent.md

```diff
@@ -364,6 +364,27 @@ The default value of the property `isChatResourceSpecificConsentEnabled` is base
 <br>
 </details>
 
+<br>
+<details>
+
+<summary><b>Configure user owner consent settings for RSC in a chat using the Graph APIs</b></summary>
+
+You can enable or disable RSC for user using Graph API. The property `isUserPersonalScopeResourceSpecificConsentEnabled` in [teamsAppSettings](/graph/api/teamsappsettings-update#example-1-enable-installation-of-apps-that-require-resource-specific-consent-in-chats-meetings) governs whether user RSC is enabled in the tenant.
+
+:::image type="content" source="../../assets/images/rsc/graph-rsc-chat-configuration.PNG" alt-text="Screenshot shows the Graph RSC team configuration.":::
+
+The default value of the property `isUserPersonalScopeResourceSpecificConsentEnabled` is based on whether [user consent settings](/azure/active-directory/manage-apps/configure-user-consent?tabs=azure-portal) is turned on or off in the tenant when RSC for user is first used. The default value is defined either when:
+
+* [TeamsAppSettings](/graph/api/teamsappsettings-get) are retrieved for the first time.
+* Teams app with RSC permissions is installed in a chat or meeting.
+
+> [!NOTE]
+> Admin control is added to allow or block RSC consent settings based on the sensitivity of the data accessed. It isn't based on the single master switch that enables or disables consent settings for app RSC permissions for all apps in the tenant.
+
+</details>
+
+<br>
+
```

Comment:
Hi @eddie-lee-msft,

I've added the steps for "Check your app for added RSC permissions for a user". While testing the GET request in the graph explorer for user we were unable to get the output for "PermissionGrants", instead we got the output for oauth2PermissionGrants. 

Request you to confirm the steps and provide your inputs.

Reply:
No @v-bvishnu, oauth2PermissionGrants is not the correct endpoint. It should be /permissionGrants. Most likely you are not able to get any output because the feature is not fully enabled worldwide.

I can provide you with sample output if you need it. Please let me know.

Reply:
@v-bvishnu here is one example:

<img width="666" alt="image" src="https://github.com/MicrosoftDocs/msteams-docs/assets/59708116/69187f1f-931a-4e24-8ace-00a643868921">